code/modules/clothing/masks/gasmask.dm

```diff
@@ -209,11 +269,13 @@
 	clothing_flags = MASKINTERNALS
 	flags_cover = MASKCOVERSEYES
 	resistance_flags = FLAMMABLE
+	max_filters = 0
```

Comment:
```suggestion
	max_filters = 1
```
Not as sure on this one. Players do like pulling a superhero gimmick with this, and since it's obviously a batman ripoff (and we do have a corresponding space suit version) it's reasonable to think they should be able to put a filter in, especially if they are going to save the station while keeping their vigilante identity.